api/v1/api.go

```diff
@@ -28,9 +33,8 @@ func NewApiV1() *apiV1 {
 // @in header
 // @name Authorization
 func (s *apiV1) RegisterRoutes(e *echo.Echo) {
-	e.GET("/status", s.handleStatus)
-
 	e.Use(AuthMiddleware)
+	e.GET("/status", s.handleStatus)
```

Comment:
I was intending to make this route un-authenticated by putting it before `e.Use(AuthMiddleware)`, but I didn't test if it worked that way - any idea?

Reply:
I'm pretty sure that would work. We do open ourselves up to anyone who wants to contentiously hit the endpoint since we don't have throttling policies in place. I doubt it is an issue at this point.